index.html

```diff
@@ -8,6 +8,12 @@
   <title>Frontend Mentor | Product preview card component</title>
 </head>
 <body>
-  <p>Hello Treasure!</p>
+  <header><img src="images/image-product-mobile.jpg" alt=""></header>
+  <p1>perfume</p1>
+  <p2>Gabrielle Essence<br>Eau De Parfum</p2>
+  <p3>A floral, solar and voluptuous<br>interpretation composed by Olivier<br>Polge, Perfumer-Creator for the House of<br>CHANEL.</p3>
```

Comment:
`p1 - p4` tags are __non-existing__ tags. Could you fix this with the appropriate tags? [Hint](https://www.w3schools.com/tags/tag_p.asp) :bulb: